sdk/tables/azure-data-tables-perf/pom.xml

```diff
@@ -21,6 +21,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
+    <checkstyle.skip>true</checkstyle.skip>
```

Comment:
Do we usually skip checkstyle for perf modules?

Reply:
Generally CI doesn't validate them but when I was running all validation they were failing